baybe/utils/sampling_algorithms.py

```diff
@@ -70,3 +76,46 @@ def farthest_point_sampling(
         remaining_point_indices.remove(selected_point_index)
 
     return selected_point_indices
+
+
+class SamplingMethod(Enum):
```

Comment:
In order to prevent some future deprecation: do you agree that it would make sense to use a more specific name here? Something like `DiscreteSamplingMethod`? Because there is also continuous sampling (an probably will be more in the future) which will likely also get its enum, and we probably don't want to mix them into one. So I think `DiscreteSamplingMethod` and `ContinuousSamplingMethod` would probably make sense. In this case, I'd say let's also kick the `for dataframes` from the docstring and just write available discrete sampling methods.

Reply:
agreed and changed